packages/web-frontend/cypress.json

```diff
@@ -5,6 +5,7 @@
   "projectId": "pty5cc",
   "requestTimeout": 30000,
   "responseTimeout": 60000,
+  "tupaia_requireNonEmptyVisualisations": false,
```

Comment:
I used the `tupaia_` prefix to namespace our cypress options

Reply:
As you can see, this option is set to `false`. The plan is:

1. Do not use this option initially, since a lot of reports will be empty
2. Manually update this option on a server and invoke the `test-e2e` script. Use the results to submit test data
3. After all reports non-empty, set this flag to `true`, making it the default behaviour moving forward

I could devise a more use friendly way of setting this off and on, but I decided the above plan is simpler and meets our current requirements.